### DISCUSSION TITLE
XYPlot reduce dot size

### DISCUSSION THREAD

**Discussion by rsand-temple** [jfree/jfreechart#285] [Q&A]
I've got a scatterplot with 20,000 points and the points all mush together because of the dot size. I've seen conversations on using custom shapes, but I just want a simple single dot. The color differentiation is enough for this purpose. Is there an easy way to tell it to use a smaller dot without defining a custom shape?

**Comment by trashgod** [comment]
A custom `Shape` is easy using `setDefaultShape()` (née `setBaseShape()`), seen [here](https://stackoverflow.com/search?tab=votes&q=%5bjfreechart%5d%20setBaseShape); see also [`ShapeUtils`](https://www.jfree.org/jfreechart/javadoc/org/jfree/chart/util/ShapeUtils.html).
